app/views/call_logs/show.haml

```diff
@@ -39,10 +39,34 @@
     .item
       %b Failure information:
       = call_log_fail_info(@log)
-%br/
-%p
-  = link_to 'Download CSV', download_details_call_log_path(@log, :format => :csv), :class => "button fimport"
 
+- if @log.recorded_audios
```

Comment:
This is should be removed, otherwise when there are no recorded audios, there will be no option to download the csv

Reply:
Sure! Good catch! Thanks!!